extensions/kafka/src/main/java/io/deephaven/kafka/KafkaTools.java

```diff
@@ -1603,15 +1603,50 @@ public Function<TopicPartition, KafkaRecordConsumer> visit(@NotNull final PerPar
      *        {@link TableDefinition}. See {@link StreamConsumerRegistrarProvider#single(SingleConsumerRegistrar)
      *        single} and {@link StreamConsumerRegistrarProvider#perPartition(PerPartitionConsumerRegistrar)
      *        per-partition}.
+     * @return the newly created KafkaIngester, the caller must call start on the returned object to begin processing
+     *         messages
      */
-    public static void consume(
+    public static KafkaIngester consume(
```

Comment:
I don't think that this is an innocuous change. It makes `KafkaIngester` part of the API. We should at least consider restricting the shutdown methods to package-private, or putting a simpler interface in front of the implementation before returning it.